README.md

```diff
@@ -1,22 +1,102 @@
-# AICoE's ArgoCD container image
+# ArgoCD deployment (v1.5.2)
+
+This repository contains the necessary artifacts to deploy ArgoCD to dev and prod.
+
+
+## Deployment Instructions
+
+### Prerequisites
+
+We use [pipenv](https://pipenv.readthedocs.io/en/latest/) to manage our
+python requirements and to ensure we're all using consistent versions of
+packages. Install pipenv by running the following command:
+
+```bash
+pip install --user pipenv
+```
+
+### Deploying to Development
+
+Run the following command from the root of this repository to deploy the
+ArgoCD project in a development environment:
+
+```bash
+pipenv shell
+pipenv install
+ansible-playbook --ask-vault-pass playbook.yaml \
+  -e kubeconfig=$HOME/.kube/config \
+  -e target_env=dev
+```
+
+### Deploying to Production
+
+Run the following command from the root of this repository to deploy the
+ArgoCD project to production:
+
+```bash
+pipenv shell
+pipenv install
+ansible-playbook --ask-vault-pass playbook.yaml \
+  -e kubeconfig=$HOME/.kube/config \
+  -e target_env=prod
+```
+
+## Dev Notes 
+
+### AICoE's ArgoCD container image
 
 This repository is containing additions to the ArgoCD container image:
 
 * KSOPS
 
-## KOPS
+### KOPS
 
 To add KOPS support we just follow [the README](https://github.com/viaduct-ai/kustomize-sops#argo-cd-integration-)
 
-## Releases
+### Releases
 
 The container image is build manually and pushed to [Quay](https://quay.io/repository/aicoe/argocd)
 
-## GCP credentials
+### GCP credentials
 
 To decrypt secrets we use some GCP key store, therefore we need to add the default application credentials that will
 enable ArgoCD to access the keystore:
 
 ```
 oc -n argocd create secret  generic  gcp --from-file=application-default-credentials=application_default_credentials.json
 ```
+
+### Creating a gpg secret
```

Comment:
How do we handle multiple keys? For example: all the Thoth inhabitants are allowed to decrypt secrets for `thoth` project in ArgoCD (and therefor deploy to all our own test project) but they are not allowed to decrypt stuff in the `production` project

Reply:
It's not handled in this PR, but I'm open to suggestions. 
One way would be to just import all these keys into argo, and store it in ansible-vault under different vault ids (test/prod/etc). This way we can limit who is able to decrypt and see the private keys based off ids.